### ISSUE TITLE
Latest Windows installer tries to install to x86 directory on 64-bit Windows

### ISSUE THREAD

**Issue by scomninos** [WinMerge/winmerge#282]
Open WinMerge-2.16.6-Setup.exe (Windows 10 Pro 64-bit)
Accept the License
Install location is shown: C:\Program Files (x86)\WinMerge

Existing installation is 2.16.4 x64 and it is located in C:\Program Files\WinMerge
I would expect this new version to install at this same location to replace/upgrade the existing one.

**Comment by sdottaka** [comment]
WinMerge-2.16.6-Setup.exe is a 32-bit installer, so it installs WinMerge in the C:\Program Files (x86)\WinMerge folder.

The 64-bit version installer is WinMerge-2.16.6-x64-Setup_no_support_for_per-user_installation.exe.

**Comment by scomninos** [comment]
Okay, I now see this file that you mention at the bottom of the Downloads page under the last section titled "Project File Releases", which is also below a section named "Unofficial Versions".  I think that the average user will simply download from one of the links at the top of the page - something above the "Other Versions" section.  (I am pretty sure that is where I had found the x64 version for v2.16.4.0 which I currently have - I did not download something that did not look like a normal release to me from the bottom of the page).

Thank you for pointing out the latest x64 installer, and for this wonderful utility.
